README.md

```diff
@@ -1 +1,39 @@
-# reservation
\ No newline at end of file
+# Project Name
+
+> Project description
+
+## Related Projects
+
+  - https://github.com/teamName/repo
```

Comment:
be sure to provide links to our other components here when the time is right 👍

Reply:
Fixed. Thanks!